docs/app/components/side-navigation/side-navigation.component.html

```diff
@@ -5,7 +5,7 @@
                 [ngModel]="versionService.version | async"
                 (ngModelChange)="versionService.setVersion($event)"
                 [btnRadio]="Version.AngularJS">AngularJS</button>
-            <button type="button" class="btn button-toggle-accent"
+            <button type="button" class="btn button-toggle-accent primary-override"
```

Comment:
Sorry, this wasn't what I meant for this change. Please bind the button class (.btn-toggle-accent) to an input property on this component. E.g. `[ngClass]="angularButtonClass"`. For the sake of consistency, do this on both buttons. The reason for this is that there are a lot of states and properties defined for buttons in the MF style, and overriding the style on the existing class would involve duplicating a lot style definitions from the quantum stylesheet.

If we used Sass then potentially we could mix in the quantum definitions, but for now I believe that this is the cleanest approach.